examples/plot_fmri_data_example.py

```diff
@@ -44,16 +37,15 @@
 from nilearn.plotting import plot_stat_map, show
 from sklearn.cluster import FeatureAgglomeration
 from sklearn.feature_extraction import image
-from sklearn.linear_model import Ridge
-from sklearn.svm import LinearSVR
 from sklearn.utils import Bunch
 
-from hidimstat.adaptative_permutation_threshold_SVR import ada_svr
 from hidimstat.clustered_inference import clustered_inference
-from hidimstat.empirical_thresholding import empirical_thresholding
 from hidimstat.ensemble_clustered_inference import ensemble_clustered_inference
-from hidimstat.permutation_test import permutation_test, permutation_test_pval
-from hidimstat.statistical_tools.p_values import pval_from_scale, zscore_from_pval
+from hidimstat.desparsified_lasso import desparsified_lasso, desparsified_lasso_pvalue
+from hidimstat.statistical_tools.p_values import zscore_from_pval
+
+# Limit the ressoruce use for the example to 5 G.
+resource.setrlimit(resource.RLIMIT_AS, (int(5 * 1e9), int(5 * 1e9)))
```

Comment:
Sorry, I'm not familiar with this. Why are you introducing that ?

Reply:
You should read the comment just before. 
It's to limit resource use, for example. This allows me to get an exception for memory usage when I run Desparsified Lasso.